facia-tool/test/public/spec/config.pinned.front.spec.js

```diff
@@ -170,4 +170,114 @@ describe('Config Pinned Front', function () {
             expect($('.cnf-front:nth(0) .input-url-path').length).toBe(1);
         }
     });
+
+    it('keeps editing existing fronts while refreshing', function (done) {
```

Comment:
My line between unit and integration test is very blurry.

What it does it load part of the tool inside a test div and then emulate events like click, type, drag.

The server is mocked so it's all in JS